eclipse-theia/theia
# vsx-registry: upgrade `sanitize-html` dependency

#### What it does


The commit updates the `sanitize-html` dependency and typings to address known _moderate_ vulnerabilities.

- [snyk: `sanitize-html` at <2.3.1](https://snyk.io/vuln/SNYK-JS-SANITIZEHTML-1070786)
- [snyk: `sanitize-html` at <2.3.2](https://snyk.io/vuln/SNYK-JS-SANITIZEHTML-1070780)

The updated version was confirmed to be _license compliant_ following our [guide](https://github.com/eclipse-theia/theia/wiki/Registering-CQs#case-new-or-updated-npm-production-dependency).


#### How to test


- start the application, and open the `extensions-view`.
- search for `theme`.
- in the list of results, click the entries to display their respective `readme`.
- the `readme` should be successfully be displayed.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
